roles/bootstrap-os/tasks/main.yml

```diff
@@ -8,4 +8,12 @@
 - include: bootstrap-centos.yml
   when: bootstrap_os == "centos"
 
-- include: setup-pipelining.yml
\ No newline at end of file
+- include: setup-pipelining.yml
+
+- name: check if atomic host
+  stat:
+    path: /run/ostree-booted
+  register: ostree
+
+- set_fact:
+    is_atomic_host: "{{ (ostree.stat.exists is defined) | ternary(true, false) }}"
```

Comment:
for this role, the fact may not be cached for future invocations. We put facts to the preinstall role as well. From that place they will be cached for all nodes.